content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -31,9 +31,9 @@ organizing your tests.
 After adding a new project, Cypress will automatically scaffold out a suggested
 folder structure. By default it will create:
 
-```text
+<!-- ```text
```

Comment:
Can you remove this if it's just going to be commented out?

Reply:
Yes, I removed it on this commit https://github.com/cypress-io/cypress-documentation/pull/4328/commits/f3b0e6a3d368318377d078945ee22fb661cbff39